### PR TITLE
Metrics app: add anomalies tab content

### PR DESCRIPTION
**Pull request by EamonnTP**
### Summary

This PR adds content for the new Anomalies tab within the create ML jobs flyout.

### Docs preview

https://observability-docs_598.docs-preview.app.elstc.co/guide/en/observability/master/inspect-metric-anomalies.html

### Related issue

Closes https://github.com/elastic/observability-docs/issues/537

### CODE REVIEW

File: docs/en/observability/inspect-metric-anomalies.asciidoc [hunk 1]
```diff
@@ -37,25 +35,40 @@ By default, the Kubernetes partition field `kubernetes.namespace` is selected.
...
-for machine type, or cloud availability zone, so that anomalies are not weighted 
+for machine type or cloud availability zone so that anomalies are not weighted 
 equally across groups.
 +
-5. Click *Enable Jobs*.
-6. You're now ready to explore your metric anomalies. Click *View anomalies* to 
-view the {ml-docs}/ml-gs-results.html[Anomaly Explorer in Machine Learning].
-
+5. By default, {ml} jobs analyze all of your metric data. You can filter to
+analyze specific metrics.
```
Comment (bmorelli25) [inline docs/en/observability/inspect-metric-anomalies.asciidoc:42]: This step just quotes the UI. Can we add an example here of what data users may want to filter on, and why? Something similar to what you did in step 4. I have no idea what that may be, but hopefully someone else tagged in this PR does 😄

**Review by bmorelli25 — APPROVED** [review]
Looks good! One comment.

File: docs/en/observability/inspect-metric-anomalies.asciidoc (hunk 1 above)
Comment (EamonnTP) [reply docs/en/observability/inspect-metric-anomalies.asciidoc]: Good point. @sorantis I added a little more context to the step. Could you give it a review? Thanks.

**Review by phillipb — APPROVED** [review]
LGTM

**Review by bmorelli25 — APPROVED** [review]

**Pull request merged**